gulp/js/default.js

```diff
@@ -56,13 +56,6 @@ var taskName = 'js',
 		compiler = webpack({
 			// Create a map of entries, i.e. {'assets/js/main': './source/assets/js/main.js'}
 			entry: helpers.webpack.getEntries(src, config.srcBase),
-			resolve: {
-				alias: {
-					handlebars: 'handlebars/runtime.js'
```

Comment:
Which is ironic since we still worked around this with `import '../../../node_modules/handlebars/dist/handlebars';`in `main.js`.